src/components/media-views.js

```diff
@@ -174,7 +163,7 @@ function createVideoTexture(url, contentType) {
       videoEl.onerror = reject;
     }
 
-    videoEl.addEventListener("loadedmetadata", () => resolve(texture), { once: true });
+    videoEl.addEventListener("canplay", () => resolve(texture), { once: true });
```

Comment:
Video size was not set yet when the `loadedmetadata` event fired for HLS videos. This is probably a more "correct" event for us to wait on for everything anyway